tests/integration/test_except.py

```diff
@@ -0,0 +1,29 @@
+def test_except_empty(c, df):
+    result_df = c.sql(
+        """
+        SELECT * FROM df
+        EXCEPT
+        SELECT * FROM df
+        """
+    )
+    result_df = result_df.compute()
+    assert len(result_df) == 0
+
+
+def test_except_non_empty(c, df):
+    result_df = c.sql(
+        """
+        (
+            SELECT 1 as "a"
+            UNION
+            SELECT 2 as "a"
+            UNION
+            SELECT 3 as "a"
+        )
+        EXCEPT
+        SELECT 2 as "a"
+        """
+    )
+    result_df = result_df.compute()
+    assert result_df.columns == "a"
+    assert set(result_df["a"]) == set([1, 3])
```

Comment:
Would you mind adding a test with NaNs? You can also use one of the prepared tables (e.g. `user_table_nan`) if it makes sense. 
It might also make sense to test this functionality against sqlite - I am just scared that especially on NULL (NaN) pandas/dask and SQL have different opinions (as I have seen so often, unfortunately...)